src/Main.elm

```diff
@@ -2,15 +2,18 @@ module Main exposing (main)
 
 import Api
 import Browser
-import Cv.Cv as Cv exposing (Cv)
+import Cv.Cv as Cv exposing (..)
+import Cv.Utdanning exposing (..)
 import Feilmelding
 import Html exposing (..)
 import Html.Attributes exposing (..)
 import Html.Events exposing (..)
 import Http
+import Melding exposing (Melding(..))
```

Comment:
Jeg skjønner ikke egentlig at dette kompilerer engang, for her importerer du `Melding(..)`, altså konstruktørene til `Melding`, men de exposes ikke fra `Melding`-modulen. Uansett så burde du fjerne `(..)` herfra.